TSPL.docc/LanguageGuide/TheBasics.md

```diff
@@ -2,25 +2,23 @@
 
 Work with common kinds of data and write basic syntax.
 
-Swift is a programming language for iOS, macOS, watchOS, and tvOS app development.
-If you have experience developing in C or Objective-C,
-many parts of Swift will be familiar to you.
+Swift is a programming language designed for developing many types of software, from mobile and desktop apps, to server, system, and embedded software.
+Many parts of Swift will be familiar if you have experience developing in other popular languages.
```

Comment:
This is redundant with the information on the AboutSwift page. Recommend deleting.

Reply:
The "Many parts of Swift …" or the "many types of software" bit? Agreed the latter is redundant.

Reply:
Wove the familiarity bit into the About Swift page instead, since it's more about the goals and design of the language, and less about introducing developers to the basics of Swift.